pyproject.toml

```
@@ -32,7 +32,7 @@ include = [
 
 [tool.poetry.dependencies]
 python = ">=3.9.2,<3.13"
-deepdiff = ">=5.5.0,!=6.0,!=6.1,<8"
+deepdiff = "^8.6.1"
```

We've been trying to start removing caret and tilde dependency specifications because they're not supported in PEP508 and we want to switch to that format at some point. 
```suggestion
deepdiff = ">=8.6.1,<9.0.0"
```